src/components/menu/test/menu.spec.ts

```diff
@@ -116,6 +116,42 @@ describe('MenuController', () => {
       expect(menu).toEqual(someMenu);
     });
 
+    it('should get the only left menu when is not rtl', () => {
+      let someMenu = mockMenu();
+      someMenu.side = 'start';
+      menuCtrl._register(someMenu);
+
+      let menu = menuCtrl.get('left');
+      expect(menu).toEqual(someMenu);
+    });
+
+    it('should get the only right menu when is rtl', () => {
```

Comment:
the platform is set to private, so no.
@brandyscarney or @jgw96, can you help on this point?